index.html

```diff
@@ -0,0 +1,16 @@
+<html>
+<head>
+  <title>
+    Pats Salmon Goods
+  </title>
+  <link href='style.css' rel='stylesheet' type='text/css'>
+</head>
+<body>
+  <div id='main'>
+    <h1> Pats Salmon Cookies </h1>
+    <img id='salmon'src='https://raw.githubusercontent.com/codefellows/seattle-201d21/master/class-06-dom-objects-domain_modeling/lab/assets/salmon.png' alt='tasty salmon'>
+  </div>
+  <script src='app.js'></script>
```

Comment:
index.html does not need the script tags because you're only making the lists on sales.html